packages/web-components/src/components/va-card/va-card.scss

```diff
@@ -0,0 +1,9 @@
+.va-card {
+  border: 1px solid var(--color-gray-medium);
+  background-color: var(--color-white);
+  padding: 1.6rem;
```

Comment:
There is no way you would have been aware of this because we have not documented it but can you convert any `rem` to `px`? 

We are managing two different library versions of USWDS right now and the body font size is set different in each of them so rem's have been really difficult for us to manage during the transition.